conf/pacific/rbd/tier_1_rbd_mirror.yml

```diff
@@ -0,0 +1,51 @@
+globals:
+  - ceph-cluster:
+     name: ceph-rbd1
+     node1:
+       role:
+          - mon
+          - mgr
+          - installer
+     node2:
+       role: client
+     node3:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node4:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node5:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node6:
+       role: rbd-mirror
+
+  - ceph-cluster:
+     name: ceph-rbd2
+     node1:
+       role:
+          - mon
+          - mgr
+          - installer
+     node2:
+       role: client
+     node3:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node4:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node5:
+       role: osd
+       no-of-volumes: 4
+       disk-size: 15
+     node6:
+       role: rbd-mirror
+
+
+
```

Comment:
Please remove empty new lines. However, it is a good practice to end with a newline character.